apps/src/templates/instructions/HeightResizer.jsx

```diff
@@ -46,25 +46,58 @@ class HeightResizer extends React.Component {
     dragStart: 0
   };
 
+  componentDidMount() {
+    this.resizerRef.addEventListener('mousedown', this.onMouseDown, {
+      passive: false
+    });
+    this.resizerRef.addEventListener('mouseup', this.onMouseUp, {
+      passive: false
+    });
+    this.resizerRef.addEventListener('mousemove', this.onMouseMove, {
+      passive: false
+    });
+    this.resizerRef.addEventListener('touchstart', this.onMouseDown, {
+      passive: false
+    });
+    this.resizerRef.addEventListener('touchend', this.onMouseUp, {
```

Comment:
I believe we have at least one helper component intended to provide a general way to bind a single handler to both mouse and touch version of events. Can we use that instead? I know there are some subtle browser differences that need to be taken into account.

Reply:
Good point.  I'll do something similar to the [VisualizationResizeBar](https://github.com/code-dot-org/code-dot-org/blob/5ab4688813269a0276b6ae944641b02be9a961b1/apps/src/lib/ui/VisualizationResizeBar.jsx) which is the horizontal bar to the right of the playspace.   It takes advantage of the mappings in [dom.js](https://github.com/code-dot-org/code-dot-org/blob/5ab4688813269a0276b6ae944641b02be9a961b1/apps/src/dom.js) while doing its own management of the listeners for reasons described in its comments.  It would be good to clean this up longer term and centralize more of this event management across our entire platform.

Reply:
Unfortunately that regressed IE11 dragging.  Given that `VisualisationResizeBar` does its own event listener management for a bunch of the events, I'd rather not copy its approach, and just keep the event listener management fully localised after all.  Given that the normal touch events are working in IE11, it seems that our special casing might have just been for IE10 and is not so important now?